src/CfSshWrapper/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CfSshWrapper")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```

Comment:
```suggestion
[assembly: AssemblyCompany("VMware")]
```
This might not be exactly right... we should match the text from the .vsix

Reply:
The VSIX manifest has "Author" = "VMware Tanzu .NET Experience" (but no "Company"). 
Reuse that for consistency?

Reply:
OK, that seems weird here... VMware should be close enough